mpsl/doc/public_mpsl.rst

```diff
@@ -0,0 +1,77 @@
+.. _external_mpsl:
```

Comment:
It would be nice to remove "public" from its name, as this does not give customers additional information. The same accounts for "_external"

Reply:
The reason is that the mpsl documentation build system separates between internal and public doc like this, and now this file is just copied without modifying it during deploy. We have a task for replacing the use of external with public. I can extend that task to also look into removing the whole prefix. And then we can fix it before next release?

Reply:
Let's aim to delete the prefix (both `public` or `external`) before the next release.